client/multi-currency/enabled-currencies-list/list-item.js

```diff
@@ -14,26 +14,33 @@ import DeleteButton from './delete-button';
 
 const EnabledCurrenciesListItem = ( {
 	// eslint-disable-next-line camelcase
-	currency: { code, flag, id, is_default, name, symbol },
+	currency: { code, flag, id, is_default, name, symbol, rate },
+	defaultCurrencyCode,
 	onDeleteClick,
 } ) => {
-	const defaultText = __( 'Default currency', 'woocommerce-payments' );
-	// eslint-disable-next-line camelcase
-	const currencyCode = is_default ? `${ code } - ${ defaultText }` : code;
-
 	const getEditUrl = ( currencyId ) => {
 		return `admin.php?page=wc-settings&tab=wcpay_multi_currency&section=${ currencyId.toLowerCase() }`;
 	};
 
+	const formattedRate = Number.parseFloat( rate ).toFixed( 2 );
```

Comment:
Should we format it differently when it's a zero-decimal currency? My understanding is that even in those cases the exchange rate could be a float.

Reply:
It [seems like](https://www.dailyfx.com/usd-jpy) it's [quite](https://www.xe.com/currencyconverter/convert/?Amount=1&From=JPY&To=USD) common to format the conversion rate as a float even when it is a zero-decimal currency, so I can't see any issue with us doing it to 2dp.